Dockerfile

```diff
@@ -2,9 +2,16 @@ FROM ruby:2.6.5-alpine AS middleman
 
 RUN apk add --update --no-cache npm git build-base
 
-ADD . .
+COPY docs/Gemfile docs/Gemfile.lock /
 
-RUN cd docs && bundle install --jobs=4 && bundle exec middleman build --build-dir=../public && cd -
+RUN bundle install --jobs=4
+
+COPY docs /docs
+COPY public /public
+COPY swagger /swagger
```

Comment:
not sure what the `COPY` in here is for really...

Reply:
The `swagger` folder has the OpenAPI spec, which gets used during the `middleman` run, so this `COPY` ensures it's there.

Reply:
tbh, I wonder if it's worth the effort. We might be able to tidy this up, but the entire goal of this stage is to generate a `public` directory that has a rendered version of the API docs, which then gets used in the final build.

This used to be `ADD . .`, but since we only use the three directories above I figured it'd be a quick win to pick them out instead of copying the entire app directory.